src/sql/src/plan/statement/dml.rs

```diff
@@ -247,6 +248,25 @@ pub fn describe_explain_plan(
     )
 }
 
+pub fn describe_explain_pushdown(
+    scx: &StatementContext,
+    statement: ExplainPushdownStatement<Aug>,
+) -> Result<StatementDesc, PlanError> {
+    let relation_desc = RelationDesc::empty()
+        .with_column("Source", ScalarType::String.nullable(false))
+        .with_column("Total Bytes", ScalarType::UInt64.nullable(false))
+        .with_column("Selected Bytes", ScalarType::UInt64.nullable(false))
+        .with_column("Total Parts", ScalarType::UInt64.nullable(false))
+        .with_column("Selected Parts", ScalarType::UInt64.nullable(false));
```

Comment:
is there prior art for column names with spaces and capital letters? Usually in the mz_internal objects at least we use `snake_case`.

Reply:
Sounds fine to me!

Reply:
I think this pre-dates my time at Materialize so I can only guess and my guess is what @bkirwi is guessing.